src/style.css

```
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.input {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-color: #fff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  height: 66px;
+  font-size: 16px;
+  font-family: inherit;
+  background-position: 26px 25px;
+  padding-left: 62px;
+  width: calc(100% - 62px);
```

```suggestion
  width: 100% 
```
Set box-sizing: border-box

---

Fix it everywhere